### DISCUSSION TITLE
Change default seed directory

### DISCUSSION THREAD

**Discussion by fnoquiq** [prisma/prisma#5674] [Q&A] [answered]
I change the schema and migrations directory (see below), but I can't change the seed. Does anyone have any ideas?

```ts
package.json
  "prisma": {
    "schema": "src/shared/prisma/schema.prisma",
    "migrateOutput": "src/shared/prisma/migrations",
    ""
  },
```

Does anyone have a listing of all the configurations that can be done by package.json?

**Comment by nikolasburk** [comment, marked as answer]
Hey @fnoquiq, this was just introduced in today's [release](https://github.com/prisma/prisma/releases/tag/2.17.0) 😄  

This should now work! If it still doesn't work for you with v2.17.0 for you, please [open a bug report for it](https://github.com/prisma/prisma/issues/new/choose) 🙂

**Reply by fnoquiq** [reply]
Yes, I saw the release. That worked!

How do I write two different seeds? One for production and one for development?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
